model/cloud.go

```diff
@@ -30,3 +30,28 @@ type StripeSetupIntent struct {
 type ConfirmPaymentMethodRequest struct {
 	StripeSetupIntentID string `json:"stripe_setup_intent_id"`
 }
+
+// Subscription model represents a subscription on the system.
+type Subscription struct {
+	ID                 string   `json:"id"`
+	CustomerID         string   `json:"customer_id"`
+	ProductID          string   `json:"product_id"`
+	AddOns             []string `json:"add_ons"`
+	StartAt            int64    `json:"start_at"`
+	EndAt              int64    `json:"end_at"`
+	CreateAt           int64    `json:"create_at"`
+	Seats              int      `json:"seats"`
+	DNS                string   `json:"dns"`
+	StripeClientSecret string   `json:"stripe_client_secret,omitempty"`
+	LicensePayload     string   `json:"license_payload"`
+	Payment            *Payment `json:"payment"`
```

Comment:
Let's omit `Payment` too because this is also related to the one time payment they have done for on-prem license purchase which also does not apply for cloud. We are working on getting the history of transactions for cloud subscriptions.